dynalab/readme.md

```diff
@@ -0,0 +1,126 @@
+# Dynalab 2.0
+
+## What is Dynalab 2.0?
+
+Dynalab 2.0 is the tool that you are going to use when you want to upload a model to a challenge on dynabench. This tool helps you to get the logic of your model and deploy it in a way the owner of the challenge can evaluate and score the performance of your algorithm.
+
+## Understand the folders
+
+### Folder structure
+
+``` none
+    └─ app
+    │  ├─ api                  <- Backend logic (Do NOT touch it)
+    │  │
+    │  ├─ resources            <- Folder to store artifacts
+    │  │   ├── tokenizer       <- artifact 1 (you can create it)
+    │  │   └── weights         <- artifact 2 (you can create it)
+    │  │
+    │  └─ domain
+    │      └── model.py        <- Script where you have to implement some methods
+    │
+    └── requirements.txt       <- File to list libraries to install
+```
+
+### Foldes contents
+
+Inside the repository you will find a folder called app. This folder contains three folders: api, domain, and resources. The api stores some logic you don't need to update. The resources folder is empty, and that's the place where you want to store your model's components (i.e. weights, tokenizers, processors, etc.). Finally, the domain folder is where you'll be including your models logic. Specifically, look for the model .py file.
```

Comment:
model .py -> model.py 
Also add:

The api folder stores certain logic that does not need to be updated and the data models used in each challenge, i.e. the input and the expected output.